cmd/kev/cmd/reconcile.go

```diff
@@ -17,34 +17,49 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 	"path"
 
 	"github.com/appvia/kev/pkg/kev"
 	"github.com/spf13/cobra"
 )
 
-func runReconcileCmd(_ *cobra.Command, _ []string) error {
+func runReconcileCmd(cmd *cobra.Command, _ []string) error {
 	cmdName := "Reconcile"
+	verbose, _ := cmd.Root().Flags().GetBool("verbose")
 
 	workingDir, err := os.Getwd()
 	if err != nil {
-		return displayError(cmdName, err)
+		return displayError(err)
 	}
 
-	manifest, err := kev.Reconcile(workingDir, os.Stdout)
+	setReporting(verbose)
+	displayCmdStarted(cmdName)
+	displayReconcileRules(verbose)
+
+	manifest, err := kev.Reconcile(workingDir)
 	if err != nil {
-		return displayError(cmdName, err)
+		return displayError(err)
 	}
 
-	os.Stdout.WriteString("...............................\n\n")
-
 	for _, environment := range manifest.Environments {
 		filePath := path.Join(workingDir, environment.File)
 		if err := writeTo(filePath, environment); err != nil {
-			return displayError(cmdName, err)
+			return displayError(err)
 		}
 	}
 
 	return nil
 }
+
+func displayReconcileRules(verbose bool) {
+	if verbose {
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2mHOW DOES RECONCILE WORK?\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ New services & volumes in a project will be added to all environments.\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ Removed services & volumes from a project will be removed from all environments.\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ Generally, updates to a project's settings DON'T affect environments.\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ Updates to a service's ports in a project DO affect environments.\n")
+		_, _ = fmt.Fprintf(os.Stdout, "\033[2m᛫ ENV VARS will be removed if they are found in an environment file but not in the project.\n")
```

Comment:
Now that I think about that I'm not sure this rule makes much sense.. Could imagine that one might add env variable in the environment specific override in special cases. It would probably complicate reconciliation so could leave as is.

Reply:
Our assumption here, is that we only allow you to override ENV VARS - not create new ones. So, as not to end in some environment variable hell. 😳

For your scenario: 
- Add the ENV VAR to your source.
- Override the target environments with the correct value.
- Override other environments with a blank value.

I should reword the rule slightly to convey what I just explained. 🙂